### PR TITLE
MTM-40524 Add a note about the processing mode note to the historic r…

### PR DESCRIPTION
**Pull request by sag-eweingaertner**
Added a note that informs the user that historic read operations will make use of the processing mode

### CODE REVIEW

**Review by torsSAG — DISMISSED** [review]

**Review by sourajitsaha17 — DISMISSED** [review]

**Review by Oezge-Akin — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by chbakar — APPROVED** [review]

**Review by Oezge-Akin — APPROVED** [review]

**Comment by BeateRixen** [comment]
@sag-eweingaertner Can I merge this? Is this only for 10.12 and later?

**Review by MWindel — APPROVED** [review]

File: content/protocol-integration/opcua-bundle/operations.md [hunk 1]
```diff
@@ -521,7 +521,7 @@ Example values to define the range for a 1D array is "0:1", for a 2D array is "0
 <td>processMappings</td>
 <td>boolean</td>
 <td>no</td>
-<td>If set to false then the read values will not be processed based on the device protocol mapping. Default is true.</td>
+<td>If set to false then the read values will not be processed based on the device protocol mapping. Default is true. *Please note:*Any data created from historic data using a device protocol will carry the same processing mode as specified in the mapping.</td>
```
[the remark this answers is not in the document]
Comment (sag-eweingaertner) [reply content/protocol-integration/opcua-bundle/operations.md]: thanks to @BeateRixen this is resolved already.

**Comment by sag-eweingaertner** [comment]
> @sag-eweingaertner Can I merge this? Is this only for 10.12 and later?

According to https://cumulocity.atlassian.net/browse/MTM-40524, the fix is in 10.11 already, so we should also merge it back to 10.11. Are you doing it? Should I?

**Comment by BeateRixen** [comment]
I will take care of merging this to develop and the release branches.

**Pull request merged**